app/src/main/java/com/goayo/debtify/model/debt/Ledger.java

```diff
@@ -0,0 +1,11 @@
+package com.goayo.debtify.model.debt;
+
+
+/**
+ * @author Gabriel Brattgård, Yenan Wang
+ * @date 2020-09-15
+ * <p>
+ * Ledger holds a list DebtTracker and manages debts and payments
+ */
+public class Ledger {
+}
```

Comment:
Both team @gabrielbrattgard @yenanw  and team @OlofSjogren @Puh00 Created a ledger class.